lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -602,6 +736,73 @@ void Caffe2ModelLoader::loadWeight(const caffe2::OperatorDef &op) {
     return;
   }
 
+  // Load tensors and quantize the values:
```

Comment:
we do not really quantize values here, right. Just loading already quantized tensors.